odoo/addons/base/ir/ir_qweb/qweb.py

```diff
@@ -555,7 +554,7 @@ def _create_def(self, options, body, prefix='fn', lineno=None):
                 arg(arg='values', annotation=None),
                 arg(arg='options', annotation=None),
                 arg(arg='log', annotation=None),
-            ], defaults=[], vararg=None, kwarg=None, posonlyargs=[], kwonlyargs=[], kw_defaults=[]),
+            ], defaults=[], vararg=None, kwarg=None, kwonlyargs=[], kw_defaults=[]),
```

Comment:
I'd think if @mart-e added the param it's necessary for 3.8 to work.

Reply:
han ouais

Reply:
Yep, at 04893ed868704d9cbf

Reply:
laissez-moi le temps d'installer 3.8 :'(